source/localizable/index.html.erb

```diff
@@ -12,6 +12,7 @@
 </head>
 
 <body>
+<!--
```

Comment:
if we aren't going to use this anymore please remove, don't just uncomment

Reply:
Also removed